bin/ci-test

```diff
@@ -10,17 +10,15 @@ function print_usage() {
 
 function handle_args() {
   export SUITE_NAME=$1
-  case $SUITE_NAME in
-    bundler1 | bundler2)
-      export MODULE=bundler
-      ;;
-    "")
-      print_usage
-      exit 1
-      ;;
-    *)
-      export MODULE=$SUITE_NAME
-  esac
+
```

Comment:
Since we are removing bundler v1 we don't need SUITE NAME. We are going export bundler as suite name.